BMPDecoder.cs

```diff
@@ -191,7 +207,7 @@ private void DecodeRGBQUAD(BinaryReader fileReader)
             quad.rgbB = Convert.ToInt16(0x00 | rgbB);
             quad.rgbG = Convert.ToInt16(0x00 | rgbG);
             quad.rgbR = Convert.ToInt16(0x00 | rgbR);
-            quad.rgbReserved = Convert.ToInt32(Convert.ToSByte(rgbReserved));
+            quad.rgbReserved = Convert.ToInt16(0x00 | rgbReserved);
```

Comment:
I have image with rgbReserved==255, and Convert.ToSByte failed for me

Reply:
Thank you, bro.
You help me a lot.